app/models/maintenance_stats/queries/github/issues_query.rb

```diff
@@ -3,7 +3,7 @@ module Queries
     module Github
       class IssuesQuery < BaseQuery
         ISSUES_QUERY = Rails.application.config.graphql.client.parse <<-GRAPHQL
-          query($owner: String!, $repo_name: String!, $open_pr_query: String!, $closed_pr_query: String!, $one_year: GitTimestamp!){
+          query($owner: String!, $repo_name: String!, $open_pr_query: String!, $closed_pr_query: String!, $one_year: DateTime!){
```

Comment:
This is the important change. The `since` parameter in the filter needed a `DateTime` object, so it was failing when receiving a `GitTimestamp`: https://docs.github.com/en/graphql/reference/input-objects#issuefilters

Reply:
🎱 Might be worth a code comment if it's not expected/obvious.